eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -247,7 +247,7 @@ public async Task<int> RunTestsAsync()
             {
                 // Timeout test run 5 minutes before the Helix job would timeout
                 var cts = new CancellationTokenSource(Options.Timeout.Subtract(TimeSpan.FromMinutes(5)));
-                var commonTestArgs = $"test {Options.Target} --logger:xunit --logger:\"console;verbosity=normal\" --blame \"CollectHangDump;TestTimeout=5m\"";                
+                var commonTestArgs = $"test {Options.Target} -s .runsettings";
```

Comment:
Can we go back to `dotnet vstest`❔ That accepts `--Settings|/Settings:<Settings File>`.

Reply:
Does it matter?

Reply:
I'm not positive but isn't `test` a larger wrapper for `vstest` w/ more moving parts❔ I was surprised we switched to `test` in #24489.

Reply:
I believe we only switched to vstest because back in the day test didn't take a dll, from other conversations I believe they both end up being wrappers for the same thing underneath the hood so it doesn't really matter...